lib/deref/hasValidParentReference.js

```diff
@@ -0,0 +1,33 @@
+var __parent = require("./../internal/parent");
+var __invalidated = require("./../internal/invalidated");
+module.exports = function fromWhenceYeCame() {
+    var reference = this._referenceContainer;
+
+    // Always true when this mode is false.
+    if (!this._fromWhenceYouCame) {
```

Comment:
Wasn't this a function? In Model line 595? Am I missing something? Is it supposed to be _allowFromWhenceYouCame?